signing_service/src/tests/tests.py

```diff
@@ -113,23 +114,24 @@ def setUp(self):
         sys.argv = sys.argv[:1]
 
     def test_that_argument_parser_should_parse_correct_input(self):
-        sys.argv += ['127.0.0.1', '1', '--concent-cluster-port', '8000']
+        sys.argv += ['127.0.0.1', '--initial_reconnect_delay', '1', '--concent-cluster-port', '8000']
```

Comment:
there should be check for `initial_reconnect_delay` as well

Reply:
:+1: done